src/components/TreeSpeciesCard.js

```diff
@@ -32,7 +32,14 @@ function TreeSpeciesCard(props) {
         justifyContent="space-between"
         alignItems="center"
       >
-        <Grid item ml={6}>
+        <Grid
+          item
+          ml={6}
+          sx={{
```

Comment:
@khalatevarun could you add some vertical padding here and undo the change other (count element) padding

Reply:
I tried doing the same initally but as a consequence of that, there is additional white space created on top and bottom of count element.
Please refer to image attached. 
![SCR-20220928-s54](https://user-images.githubusercontent.com/68511458/192810134-3d970809-97d3-47ea-991e-e5a2371ffd2d.png)

Reply:
I think adding padding for the count element is would be better as it would result only slight increase in the list height.

Reply:
We could fix this by adding `align-self: stretch` to the count box.

Reply:
If we don't add the padding to the description side of the box the spacing will not look the same for all cards
![image](https://user-images.githubusercontent.com/31519867/192978081-961303db-8acc-4b64-9ef9-b7a844daaa97.png)

Reply:
makes sense, but it would result in cards of different heights since some name will take 2 lines and adding vertical padding will further increase vertical height thus resulting in inconsistency?

Can surely make changes suggested by you if the above mentioned concern is not a big deal. 
Let me know if I should undo my changes and add vertical padding.

Reply:
I think its fine if height increases when we have more than 1 line.

Reply:
okay, will update changes